src/extension/content-script/setLightningData.ts

```diff
@@ -3,6 +3,15 @@ import utils from "~/common/lib/utils";
 import { Battery } from "~/types";
 
 const setLightningData = (data: [Battery]): void => {
+  // Pass lightningData to e2e tests
+  if (process.env.NODE_ENV == "development") {
```

Comment:
This will also be triggered when just developing locally? If yes, can we try to avoid it?

Reply:
Yes it will. I don't see why it should be avoided since this is part of the testing that is useful also to test local changes

Reply:
> also to test local changes

Can you document how this would be done? I.e. here: https://github.com/getAlby/lightning-browser-extension#e2e-tests-via-playwright-using-testing-library

- add new headline
- explain that we have special code in `development` to test batteries
- how to use this locally

For people looking a this the first time it might be hard to understand how and when to use this.

Reply:
Are you sure i should document it in the readme? The only meaningful way to use this is with the [getLightningData](https://github.com/getAlby/lightning-browser-extension/pull/1041/files#diff-800d7b264f1edb5e8353d2c7aac0efb98b2169ed6709e7b0cc2fdb89f09acea4R122-R137) in testBatteries.spec.ts, and it is just the standard js cross-origin communication that is used by the extension everywhere to communicate with the background scripts.

To add a test here you only need to change [the Unit object on the start of the script](https://github.com/getAlby/lightning-browser-extension/pull/1041/files#diff-800d7b264f1edb5e8353d2c7aac0efb98b2169ed6709e7b0cc2fdb89f09acea4R4-R32) there is no need to change the code normally.

I think you are misunderstanding this as a way for the developer to debug the lightning data in realtime during the development but this is limited to this specific test. If not, do you have an idea on how to document this in a way that is not confusing for someone that is not interested in the internal functions working?